### PR TITLE
[5.0] ModalField Fix missing icon for select btn

### PR DESCRIPTION
**Pull request by Fedik**
### Summary of Changes

Fix missing icon for select btn 

### Testing Instructions

Go to menu, edit menu Single Article, check "select article" field

### Actual result BEFORE applying this Pull Request
Select button without icon
Screenshot 2023-09-14_15-21-24

### Expected result AFTER applying this Pull Request
Select button with icon
Screenshot 2023-09-14_15-21-07

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed
- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: layouts/joomla/form/field/modal-select/buttons.php [hunk 1]
```diff
@@ -74,7 +74,7 @@
 <button type="button" class="btn btn-primary" <?php echo $value && !$isSelectAlways ? 'hidden' : ''; ?>
         data-button-action="select" <?php echo !$isSelectAlways ? 'data-show-when-value=""' : ''; ?>
         data-modal-config="<?php echo $this->escape(json_encode($modalSelect)); ?>">
-    <span class="<?php echo $buttonIcons['select'] ?? 'icon-file'; ?>" aria-hidden="true"></span> <?php echo Text::_('JSELECT'); ?>
+    <span class="<?php echo !empty($buttonIcons['select']) ? $buttonIcons['select'] : 'icon-file'; ?>" aria-hidden="true"></span> <?php echo Text::_('JSELECT'); ?>
```
Comment (Quy) [inline layouts/joomla/form/field/modal-select/buttons.php:77]: `?:` should work too??

Comment (Fedik) [reply layouts/joomla/form/field/modal-select/buttons.php:77]: The array may be empty

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on ff1009775d562cc3610247605191826f624af071<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41750">issues.joomla.org/tracker/joomla-cms/41750</a>.</sub>

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on ff1009775d562cc3610247605191826f624af071<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41750">issues.joomla.org/tracker/joomla-cms/41750</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41750">issues.joomla.org/tracker/joomla-cms/41750</a>.</sub>

**Pull request merged**